src/qforte/utils/exponentiate.py

```diff
@@ -0,0 +1,74 @@
+"""
+Functions for exponentiation of qubit operator terms (circuits)
+"""
+
+import qforte
+import numpy
+
+
+def exponentiate_single_term(param, term):
+    """
+    A function which returns an exponentiated version of a single (param * term)
+    from a parameterized qubit state preparation circuit.
+
+    :param term: the term to be exponentiated
+    """
+
+    # TODO: look into multiplication by 1.0j? (Nick)
+    # TODO: This code is very similar to that used in PyQuil,
```

Comment:
@fevangelista, what should we do given that this is a similar patch of code to that found in PyQuil? Reference their code? or change ours?